backends/p4tools/testgen/options.h

```diff
@@ -57,6 +57,9 @@ class TestgenOptions : public AbstractP4cToolOptions {
     /// String of selected branches separated by comma.
     std::string selectedBranches;
 
+    /// String of a pattern for resulting tests.
+    std::string pattern;
```

Comment:
Only a small detail, which I can adjust later in the PR with an exploration strategy for reachability - it'd be nice to specify how we want this pattern and add something in the REAME as well.